### ISSUE TITLE
InfluxDB 2.0 integration

### ISSUE THREAD

**Issue by webnar** [xirixiz/dsmr-reader-docker#216]
At his point, it is not yet possible (I couldn't find it anywhere) to connect the InfluxDB 2.0.
Is it possible to integrate this in the DSMR reader docker?

**Comment by dennissiemensma** [comment]
Hi, it's scheduled in https://github.com/dsmrreader/dsmr-reader/issues/1380 for DSMR-reader v5.x

**Comment by webnar** [comment]
Sounds good, any estimation when it will be released?

**Comment by dennissiemensma** [comment]
It will take quite some while, since it's not backwards compatible so it relies on DSMR-reader v5 first, see the issue comments as well.

**Comment by webnar** [comment]
Is it already possible to upgrade to DSMR-reader v5?

**Comment by dennissiemensma** [comment]
@webnar unfortunately there has been some delay in the release of v5.

One of the last (and more large) technical changes planned for v5 ([ #1202 ](https://github.com/dsmrreader/dsmr-reader/issues/1202)) did not seem to work properly on every test-upgrade, so I will need to either rework it or revert it to the old situation first.

**Comment by dennissiemensma** [comment]
I've decided to revert the https://github.com/dsmrreader/dsmr-reader/issues/1202 issue from the v5 release. It does mean I will need to revert a lot of scripts and documentation that was updated though, but it's for the best. I'm considering only supporting Docker installations next year when I'll bump to v6, which should make these kinds of upgrades way more easy.

For now I will go back to the old situation and will also need to include https://github.com/dsmrreader/dsmr-reader/issues/1210 as well.
